elated-egrets/bangazon-internal-site
# Pwm faker

## DESCRIPTION

## How to pull down
Commands

```
git fetch --all
git checkout pwm-faker
```

## Steps to test

1. Start virtual environment
1. Make sure Django server is running
1. ...

## What you changed
created directory management
created directory commands inside management
added faker_factory.py inside commands
**details of what to do are in comments of faker_factory.py but here they are as well**

*inside of root directory exists django_data.sh file. For Mac users, save this file as django_data.sh in **usr/local/bin** (Which is at your machine's root, not your Users folder)

 You will need to change the permissions of this file with chmod +x django_data.sh to make it executable.

Run this command in project:
`django_data.sh employee_portal faker_factory`
(this will handle all migrations commands as you can see in django_data.sh file so no need to run python manage.py makemigrations etc.)

this will create and populate database using models
this command will also delete and rebuild database each time it is run

## What was added to read me

## Final Checklist
Before merging.
1. Has each file changed been reviewed?
2. Has the readme been updated to reflect changes made?
3. Has a teammate approved the PR?

Then Ship It!
